pkg/testframework/console.go

```diff
@@ -0,0 +1,162 @@
+package testframework
```

Comment:
Thinking about this now, it might be nice to split it up.  Right now it seems to be all the funcs needed just to test management state.  I'd prob rather separate out feature tests than keep adding to this one file:

```bash
/pkg
   |_ testframework
        |_ console
            |_ managementstate.go
            |_ otherfeature.go
            |_ yetanotherfeature.go
        |_ deployment
        |_ etc
```
Thoughts?

Reply:
Makes sense to split it here, but Im not sure what the otherfeatures will be. The ones that pop to my head are, to start/stop the operator by scaling it up and down, but that can be on a `/pkg/testframework/operator.go` file.
Not sure what else at this state of the console-operator we should test.

Reply:
Oh, we will probably test the top level `console` config, which has, for example, `LogoutRedirect`. Also the `Operator` config itself has `Branding`, `DocumentationBaseURL`, as well as a `NodeSelector` override.  

Eventually we may support custom `Route` or perhaps a set of `Routes` (in discussion).